app/views/puppetclasses/_class_parameters.html.erb

```diff
@@ -1,18 +1,62 @@
-      <td><%= obj.class.model_name == "Host" ? host_key_with_diagnostic(obj, value_hash, key) : hostgroup_key_with_diagnostic(obj, key)%></td>
-      <td>
-          <%= link_to_function(_("override"), "override_class_param(this)", :title => _("Override this value"),
-                               :'data-tag' => 'override', :class =>"btn btn-default") if authorized_via_my_scope("host_editing", "create_params") && !key.overridden?(obj) %>
-      </td>
+<div id='<%= puppetclass.to_s.gsub(':', '_') + '_' + puppetclass.id.to_s %>' class='tab-pane fields'>
+  <% lookup_keys = overridable_lookup_keys(puppetclass, obj) %>
+  <% value_hash = Classification::ClassParam.new(:host => obj).inherited_values if obj.class.model_name == "Host" %>
+  <table class="table table-condensed">
+    <thead>
+    <tr>
+      <th class='col-md-2'><%= _('Name') %></th>
+      <th class='col-md-7'><%= _('Value') %></th>
+      <th><%= _('Actions') %></th>
     </tr>
-<% end %>
+    </thead>
+    <tbody>
+    <% lookup_keys.each_with_index do |key, i| %>
+        <tr class="lookup_key_row <%="overridden" if key.overridden?(obj) %>" data-lookup-key-id="<%=key.id%>">
+          <td class='col-md-2'>
+            <%= content_tag :span, key.key, :class => "col-md-2 lookup_key_name", :'data-property' => 'name' %></td>
+          <td class='col-md-7'><%= obj.class.model_name == "Host" ? host_key_with_diagnostic(obj, value_hash, key) : hostgroup_key_with_diagnostic(obj, key) %></td>
+          <td>
+            <%= link_to(_("override"), nil, :title => _("Override this value"),
+                                 :'data-tag' => 'override', :class => "btn btn-default override_button" ) if authorized_via_my_scope("host_editing", "create_params") %>
+            <%= content_tag :span, _("Overridden"), :class => "overridden_message" %>
+          </td>
+        </tr>
+        <tr id="lookup_value_for_<%=key.id%>" class="lookup_key_override_row <%="visible" if key.overridden?(obj) %>">
+          <% value = key.overriding_value(obj) %>
+          <td class='col-md-2'>
+            <%= content_tag :span, key.key, :class => "col-md-2 lookup_key_name", :'data-property' => 'name' %></td>
+          <td class='col-md-7'>
+              <%= hidden_field_tag "host[lookup_values_attributes][#{key.id}][lookup_key_id]", key.id %>
+              <%= hidden_field_tag("host[lookup_values_attributes][#{key.id}][id]", value.id) if value.present? %>
+              <%= hidden_field_tag "host[lookup_values_attributes][#{key.id}][_destroy]", value.blank?, :class => "destroyer" %>
+              <%= text_area_tag("host[lookup_values_attributes][#{key.id}][value]", value.try(:value),
+                            :class => "overriding-value col-md-5",
+                            :rows => 1,
+                            :placeholder => _("Value")) %>
+              <span class="hidden_value">
+                <%= check_box_tag "host[lookup_values_attributes][#{key.id}][hidden]" %>
```

Comment:
Does this do anything yet?  I don't think it's on a lookup value, it might be better as an attribute on the lookup key rather than values.